apps/examples/testcase/Kconfig

```diff
@@ -26,6 +26,7 @@ source "$APPSDIR/examples/testcase/ta_tc/wifi_manager/utc/Kconfig"
 source "$APPSDIR/examples/testcase/ta_tc/wifi_manager/itc/Kconfig"
 source "$APPSDIR/examples/testcase/ta_tc/mqtt/utc/Kconfig"
 source "$APPSDIR/examples/testcase/ta_tc/mqtt/itc/Kconfig"
+source "$APPSDIR/../os/test_fw/Kconfig"
```

Comment:
I think this is not good. Location of Kconfig should be in os, not apps because test_fw is a kernel module.

Reply:
Ok.. I changed it to os/Kconfig and sourced it under kernel features